### PR TITLE
Remove Beta label from Reports endpoints

### PR DESCRIPTION
**Pull request by eoin-shaughnessy-cko**
# Description of changes in PR

[//]: # 'Please add your description here'

## Checklist

- [ ] Added a changelog entry to the `changelog.md` file in either `abc_spec` or `nas_spec`
- [ ] Contacted the Tech Docs team about any corresponding guides that need to be updated for www.checkout.com/docs

**Contributing options**: Look at [our documentation](https://checkout.atlassian.net/wiki/spaces/PD/pages/2169506663/API+ref+publication+process) for options to contribute to the API reference.

### CODE REVIEW

**Comment by eoin-shaughnessy-cko** [comment]
We will merge these changes for our December 7th release, as per [stakeholder request](https://checkout.slack.com/archives/C3RAJ3T42/p1668446391235239).

Couple of questions...

Why were the `package-lock.json` and `yarn.lock` files included when I didn't edit them?

Will we need to update the [NAS Changelog](https://github.com/checkout/checkout-api-reference/blob/b4594e224dc4994e494dcc4fc3022058f7411697/nas_spec/changelog.md) at the same time we publish the changes here?

**Comment by eoin-shaughnessy-cko** [comment]
We will merge these changes for our December 7th release, as per [stakeholder request](https://checkout.slack.com/archives/C3RAJ3T42/p1668446391235239).

Couple of questions...

Why were the `package-lock.json` `package.json`and `yarn.lock` files included when I didn't edit them?

Will we need to update the [NAS Changelog](https://github.com/checkout/checkout-api-reference/blob/b4594e224dc4994e494dcc4fc3022058f7411697/nas_spec/changelog.md) at the same time we publish the changes here?

**Comment by eoin-shaughnessy-cko** [comment]
Tried rerunning Vercel check a few times, but still failing...

**Comment by eoin-shaughnessy-cko** [comment]
@gareth-forshaw-cko Thanks for reviewing the [PR to remove the Beta label from _Reporting_ > _Reports API_ on the main site](https://github.com/cko-web/checkout-docs-site/pull/1773)!

Can I also get a review on this related PR, about removing the Beta label from parts of the API Ref? :)

File: package-lock.json [hunk 1]
```diff
@@ -1740,23 +1739,6 @@
       "resolved": "https://registry.npmjs.org/dompurify/-/dompurify-2.3.3.tgz",
       "integrity": "sha512-dqnqRkPMAjOZE0FogZ+ceJNM2dZ3V/yNOuFB7+39qpO93hHhfRpHw3heYQC7DPK9FqbQTfBKUJhiSfz4MvXYwg=="
     },
-    "node_modules/dotnet-sdk-3.1": {
```
Comment (gareth-forshaw-cko) [inline package-lock.json:1743]: I don't think we should commit this file. This is what stopped Vercel from running.

**Review by gareth-forshaw-cko — CHANGES_REQUESTED** [review]
The actual changes look good. Just need to drop those two Java files and then Vercel should build and I will approve.

**Comment by gareth-forshaw-cko** [comment]
> 

Sorry, just saw this comment now, Those are the two/three files we had to edit to get our local environments/builds running. So we just need to exclude them when we commit as they don't need to go to the server, those changes are just for our local laptop environments to work.

Not 100% sure about the change-log, looks like it is mostly dev changing that so I think for the scope of this change it may not be needed.

**Comment by eoin-shaughnessy-cko** [comment]
@gareth-forshaw-cko Cheers! I removed those files by running `git checkout master -- FILENAME` in my command line.

**Review by gareth-forshaw-cko — CHANGES_REQUESTED** [review]
Good news is Vercel is fixed, bad news is that I think we still need to remove the yarn.lock file from this PR.

**Review by gareth-forshaw-cko — APPROVED** [review]

**Pull request merged**